module/documents/item.mjs

```diff
@@ -784,25 +795,21 @@ export default class Item5e extends Item {
       "flags.dnd5e.use": {type: this.type, itemId: this.id, itemUuid: this.uuid}
     }, options);
 
-    // Reference aspects of the item data necessary for usage
-    const resource = is.consume || {};        // Resource consumption
-    const isSpell = item.type === "spell";    // Does the item require a spell slot?
-    const requireSpellSlot = isSpell && (is.level > 0) && CONFIG.DND5E.spellUpcastModes.includes(is.preparation.mode);
-
     // Define follow-up actions resulting from the item usage
-    config = foundry.utils.mergeObject({
-      createMeasuredTemplate: item.hasAreaTarget,
-      consumeQuantity: is.uses?.autoDestroy ?? false,
-      consumeRecharge: !!is.recharge?.value,
-      consumeResource: !!resource.target && (!item.hasAttack || (resource.type !== "ammo")),
-      consumeSpellLevel: requireSpellSlot ? is.preparation.mode === "pact" ? "pact" : is.level : null,
-      consumeSpellSlot: requireSpellSlot,
-      consumeUsage: !!is.uses?.per && (is.uses?.max > 0)
-    }, config);
-
-    // Display a configuration dialog to customize the usage
-    if ( config.needsConfiguration === undefined ) config.needsConfiguration = config.createMeasuredTemplate
-      || config.consumeRecharge || config.consumeResource || config.consumeSpellSlot || config.consumeUsage;
+    if ( config.consumeSlotLevel ) {
+      console.warn(`You are passing 'consumeSlotLevel' to the ItemUseConfiguration object, which now expects a key as 'slotLevel'.`);
+      config.slotLevel = config.consumeSlotLevel;
+      delete config.consumeSlotLevel;
+    }
+    if ( Number.isNumeric(config.slotLevel) ) {
+      console.warn(`You are passing a numeric value to 'slotLevel' which expects a key from Actor5e#system#spells.`);
+      config.slotLevel = `spell${config.slotLevel}`;
+    }
+
+    config = foundry.utils.mergeObject(this._getUsageConfig(), config);
+
+    // Are any default values necessitating a prompt?
+    const needsConfiguration = Object.values(config).includes(true);
```

Comment:
That is correct, yes. Since we will be using `null` for "the item is unable to do this", the follow-up 'prompt' work should set it to `false` (since it is able, but should not ask for it).